### ISSUE TITLE
external resolver

### ISSUE THREAD

**Issue by rfrancis** [vic/asdf-ocaml#1]
Is there any hope for also having the plugin build and install aspcud or in general some external resolver?  OPAM is barely useful without one -- the very first package I wanted to install with it, install claims it isn't there although I can do info etc on it just fine.  Apparently this happens because the internal resolver is junk.

I'd just install one of them with yum but, ha ha, none of them are in the repos for Fedora or RHEL. And really, it'd be nice to have it all packaged together here, if it's possible.

**Comment by vic** [comment]
Yep it would be possible, I'm not sure if also installing an external resolver should be the default behaviour. This week I wont have much time to look at how complicated it'd be to also install a resolver along side opam, if you do have time, feel free to submit a PR is this is bothering you much, just be sure to enable it via an environment variable, say `env ASDF_OCAML_RESOLVER=aspcud asdf install ocaml` in which case the plugin would also install aspcud.

**Comment by rfrancis** [comment]
I dunno that I feel all that strongly (and I went ahead and installed it by
hand for my own situation), but I can't really resist a "feel free to
submit a PR", so I'll have a look, sure. :-)

**Comment by rfrancis** [comment]
Okay, I looked. :-)  Never mind.  Complicated enough to be undesirable, I think.  (Really, as soon as I had to have it getting source from Sourceforge...)
